src/api.ts

```diff
@@ -1,6 +1,23 @@
 import { multinetApi } from 'multinet';
 import { host } from '@/environment';
+import { UserInfo } from '@/types';
 
 const api = multinetApi(`${host}/api`);
 
+export async function getUserInfo(): Promise<UserInfo | null> {
+  const resp = await fetch(`${host}/user/info`, {
+    credentials: 'include',
+  });
+
+  if (resp.ok) {
+    return resp.json();
+  }
+
+  return null;
+}
+
+export function logout() {
+  return fetch(`${host}/user/logout`);
+}
+
```

Comment:
No, that issue is fairly orthogonal to what's going on here (it will just add convenience functions we'll be able to call, similar to the rest of the multinetjs library). In this case, it's temporary until we can close https://github.com/multinet-app/multinet-server/issues/378--because the `/user` API doesn't sit at the same prefix as everything else currently, we can't just use the API object to access it.

Reply:
Makes sense. Thanks for the clarification